scripts/plugins.py

```diff
@@ -45,6 +48,8 @@
         Deploy_Plugins()
     elif mode == "install":
         Install_Plugins()
+    elif mode == "schema_parsing":
```

Comment:
I think the naming of the command line arg could be improved. 

Off the top of my head; 
- Schema
- Parser
- DeCLI

Reply:
reason for picking this is we are only invoking the schema parsing function. So certainly DeCLI and schema would be confusing.